ui/tests/unit/models/generated-item-test.js

```diff
@@ -0,0 +1,18 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import { module, test } from 'qunit';
+
+import { setupTest } from 'vault/tests/helpers';
+
+module('Unit | Model | generated item', function (hooks) {
+  setupTest(hooks);
+
+  test('it exists', function (assert) {
+    const store = this.owner.lookup('service:store');
+    const model = store.createRecord('generated-item', {});
+    assert.ok(model);
```

Comment:
Can you add a message if it passes? Helps for debugging (I know there's only one test in this file, but in case more are added. Something like:
```suggestion
    assert.ok(model, "model of generated-item exists");
```